Uppsala/UppEt41.xml

```diff
@@ -0,0 +1,502 @@
+                            <bibl><ptr target="bm:Loefgren1929Evangeliska"/><citedRange unit="page">20</citedRange>
+                                <citedRange unit="number">Nachtrag 7</citedRange></bibl> based on information provided by 
+                            Jonas Iwarson, who at that time was the
+                            head of the Swedish Mission Station in <placeName ref="LOC1579Asmara"/>, 
+                            the manuscript was located in the library of the Mission.
+                        </provenance>
+                        
+                        <acquisition>According to <ref target="#e8"/>, the manuscript reached its current repository in 
+                            <date when="1968">1968</date>.
+                        </acquisition>
+                    </history>
+
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Loefgren1974Katalog"/>
+                                            <citedRange unit="page">126-129</citedRange>
+                                            <citedRange unit="number">37</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Loefgren1929Evangeliska"/>
+                                            <citedRange unit="page">20</citedRange>
+                                            <citedRange unit="number">Nachtrag 7</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:alvin"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:BmWebsite"/>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    
+                </msDesc>
+
+            </sourceDesc>
+        </fileDesc>
+        
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+            <tagsDecl>
+                <rendition xml:id="partialRubric">Red ink is used partially in the rubrication of punctuation.</rendition>
```

Comment:
`tagsDecl `is about the tags used in the encoding, and rendering is a technical thing that can be applied to stylesheets, see examples here
https://www.tei-c.org/release/doc/tei-p5-doc/en/html/examples-tagsDecl.html

I guess you mean `<seg type="rubrication">` in handDesc

Reply:
This had been developed to encode the difference between completely rubricated text and partially rubricated elements (e.g. in punctuation elements)
https://betamasaheft.eu/Guidelines/?q=partial&id=text-encoding

Reply:
> This had been developed to encode the difference between completely rubricated text and partially rubricated elements (e.g. in punctuation elements) https://betamasaheft.eu/Guidelines/?q=partial&id=text-encoding

Yes, exactly, I was intending to accommodate for encoding "partial rubrication"

Reply:
I would suggest being less creative with generic TEI guidelines. 
`tagsDecl` should not be used for this. We can introduce a subtype `partial`, but so far in most cases this was described in `rubricatiojn`.

Reply:
This is included in the guidelines and we have used it for several years. I have no opinion on the encoding itself and would pobably also have preferred a subtype attribute, but now I would rather not go over the old records (if a hiwi can do it, great!)

Reply:
https://github.com/BetaMasaheft/Documentation/issues/1425

Reply:
Thank you ) sorry for confusion.